parquet-encoding/src/main/java/org/apache/parquet/column/values/bitpacking/BytePacker.java

```diff
@@ -71,6 +73,9 @@ public final int getBitWidth() {
    * @param output the output values
    * @param outPos where to write to in output
    */
+  public abstract void unpack8Values(final ByteBuffer input, final int inPos, final int[] output, final int outPos);
+  
+  //Compatible API
   public abstract void unpack8Values(final byte[] input, final int inPos, final int[] output, final int outPos);
```

Comment:
nvm just noticed the comment below, will move the implementation of the method that is currently hard-coded in the generation into this abstract class.